File: exs/mundo_1/c#/002.cs
// crie um script que leia o dia, o mês e o ano de nascimento de uma pessoa e mostre uma mensagem com a data formatada 

using System;

class Program {
    // criado para facilitar a entrada de dados, baseado no input do python
    // static serve para ser lido sem instância de classe. é importante, recomendo estudar
    public static string input(string text) {
        Console.Write(text);
        return Console.ReadLine();
    }
    
    // removido o "string[] args" por não ser necessário
    public static void Main() {
        int dia = int.Parse(input(
            "Qual o dia do seu nascimento? "
        ));
        int mes = int.Parse(input(
            "E o mês? [0-12] "
        ));
        int ano = int.Parse(input(
            "Por fim, qual o ano? "
        ));
        
        Console.WriteLine($"Você nasceu em {dia}/{mes}/{ano}");
    }
}

// int.Parse transforma uma string em inteiro, outros tipos tem o mesmo método como:  float.Parse, doble.Parse
// para transformas de volta em string pode se usar variável.ToString

File: exs/mundo_1/java/002.java
/*
Desafio 002
Problema: Faça um programa que leia o nome de uma pessoa e mostre
        uma mensagem de boas-vindas.
Resolução do problema:
*/
import java.util.Scanner;

class Main{
    public static void main(String[] args){
        Scanner scan = new Scanner(System.in); // Input
        System.out.print("Digite seu nome: ");
        String nome = scan.nextLine(); // armazena o input a variavel 'nome'
        System.out.printf("Prazer, %s%n", nome);
    }
}
